amplitude/settings.py

```diff
@@ -11,6 +11,7 @@
 if not isinstance(IGNORE_URLS, list):
     error = '"AMPLITUDE_IGNORE_URLS" must be a list of URLs or URL names'
     raise ImproperlyConfigured(error)
+MIN_ID_LENGTH = getattr(settings, 'AMPLITUDE_MIN_ID_LENGTH', None)
```

Comment:
Could we please make sure that `MIN_ID_LENGTH` in an integer?

```suggestion
MIN_ID_LENGTH = getattr(settings, 'AMPLITUDE_MIN_ID_LENGTH', None)
if MIN_ID_LENGTH and not isinstance(MIN_ID_LENGTH, int):
    raise ImproperlyConfigured('"AMPLITUDE_MIN_ID_LENGTH" must be an integer')
```

Then for a test in `test/settings.py`
```
def test_min_id_length(settings):
    settings.AMPLITUDE_MIN_ID_LENGTH = 'test'

    with pytest.raises(ImproperlyConfigured) as error:
        from amplitude import settings as appsettings
        reload(appsettings)
    error.match('"AMPLITUDE_MIN_ID_LENGTH" must be an integer')
```